### DISCUSSION TITLE
能否增加其他模型

### DISCUSSION THREAD

**Discussion by TsirTV** [BurnHair/docs#173] [功能建议]
最近看claude 3.5被捧上天了，不知道能不能增加这个模型，想试试。谢谢。

**Comment by NoahBishop** [comment]
这是open-ai 的模型，肯定没啊

**Comment by BennyThink** [comment]
以后会考虑的 :-)

**Reply by hijos** [reply]
> 以后会考虑的 :-)

能否增加最新的 gpt-4o-2024-08-06 模型

**Comment by BennyThink** [comment]
https://github.com/orgs/BurnHair/discussions/207

好了，并且在考虑 Mistral AI
